Softeq.NetKit.Chat.SignalR/Hubs/HubEvents.cs

```diff
@@ -28,6 +28,10 @@ public static class HubEvents
         public const string ExceptionOccurred = "ExceptionOccurred";
         public const string RequestSuccess = "RequestSuccess";
 
+        public const string DirectMessageAdded = "DirectMessageAdded";
```

Comment:
We have agreed that we will notify client using the same event types, so you already have it.